frontend/app/controllers/Joiner.scala

```diff
@@ -138,6 +138,7 @@ trait Joiner extends Controller {
         eventId <- PreMembershipJoiningEventFromSessionExtractor.eventIdFrom(request)
         event <- EventbriteService.getBookableEvent(eventId)
       } yield {
+        EventbriteService.getService(event).wsMetrics.put("user-sent-to-eventbrite", 1)
```

Comment:
Not a bad idea – a followup task I'm trying to have a go at now is tracking when a user signs up (which we do already) but also recording whether they signed up for a Guardian Live event, a Masterclass, or just a plain old member. 

The main compelling argument for using the same metric name is that both instances still require another click from the user (eg. the big green Order Now button) so there's not much functional difference between this experience and being on EventBrite.